test/train-sets/ref/automl_readable_qc.txt

```diff
@@ -0,0 +1,3345 @@
+_automl_config_manager_configs[159]_exclusions[0].size() = 3
+_automl_config_manager_configs[159]_exclusions[0][0] = 83
+_automl_config_manager_configs[159]_exclusions[0][1] = 84
+_automl_config_manager_configs[159]_exclusions[0][2] = 84
+_automl_config_manager_configs[159]_lease = 4000
+_automl_config_manager_configs[159]_state = New
+_automl_config_manager_configs[159]_type = Interaction
+_automl_config_manager_configs[160]_exclusions.size() = 1
+_automl_config_manager_configs[160]_exclusions[0].size() = 3
+_automl_config_manager_configs[160]_exclusions[0][0] = 83
+_automl_config_manager_configs[160]_exclusions[0][1] = 84
+_automl_config_manager_configs[160]_exclusions[0][2] = 105
+_automl_config_manager_configs[160]_lease = 4000
+_automl_config_manager_configs[160]_state = New
+_automl_config_manager_configs[160]_type = Interaction
+_automl_config_manager_configs[161]_exclusions.size() = 1
+_automl_config_manager_configs[161]_exclusions[0].size() = 3
+_automl_config_manager_configs[161]_exclusions[0][0] = 83
+_automl_config_manager_configs[161]_exclusions[0][1] = 105
+_automl_config_manager_configs[161]_exclusions[0][2] = 106
+_automl_config_manager_configs[161]_lease = 4000
+_automl_config_manager_configs[161]_state = New
+_automl_config_manager_configs[161]_type = Interaction
+_automl_config_manager_configs[162]_exclusions.size() = 1
+_automl_config_manager_configs[162]_exclusions[0].size() = 3
+_automl_config_manager_configs[162]_exclusions[0][0] = 84
+_automl_config_manager_configs[162]_exclusions[0][1] = 84
+_automl_config_manager_configs[162]_exclusions[0][2] = 84
+_automl_config_manager_configs[162]_lease = 4000
+_automl_config_manager_configs[162]_state = New
+_automl_config_manager_configs[162]_type = Interaction
+_automl_config_manager_configs[163]_exclusions.size() = 1
+_automl_config_manager_configs[163]_exclusions[0].size() = 3
+_automl_config_manager_configs[163]_exclusions[0][0] = 84
+_automl_config_manager_configs[163]_exclusions[0][1] = 105
+_automl_config_manager_configs[163]_exclusions[0][2] = 106
+_automl_config_manager_configs[163]_lease = 4000
+_automl_config_manager_configs[163]_state = New
+_automl_config_manager_configs[163]_type = Interaction
+_automl_config_manager_configs[164]_exclusions.size() = 1
+_automl_config_manager_configs[164]_exclusions[0].size() = 3
+_automl_config_manager_configs[164]_exclusions[0][0] = 105
+_automl_config_manager_configs[164]_exclusions[0][1] = 105
+_automl_config_manager_configs[164]_exclusions[0][2] = 105
+_automl_config_manager_configs[164]_lease = 4000
+_automl_config_manager_configs[164]_state = New
+_automl_config_manager_configs[164]_type = Interaction
+_automl_config_manager_configs[165]_exclusions.size() = 1
+_automl_config_manager_configs[165]_exclusions[0].size() = 3
+_automl_config_manager_configs[165]_exclusions[0][0] = 105
+_automl_config_manager_configs[165]_exclusions[0][1] = 105
+_automl_config_manager_configs[165]_exclusions[0][2] = 106
```

Comment:
@bassmang should we just save the configs being used and re-generate from the champ instead of storing all of them? this might get out of control

Reply:
unrelated to this PR but open question

Reply:
Not sure if this is possible, especially since randomness is used to generate challengers